packages/core/admin/ee/admin/src/pages/SettingsPage/pages/AuditLogs/components/TableRows.tsx

```diff
@@ -3,17 +3,38 @@ import React from 'react';
 import { Flex, IconButton, Tbody, Td, Tr, Typography } from '@strapi/design-system';
 import { onRowClick, stopPropagation } from '@strapi/helper-plugin';
 import { Eye } from '@strapi/icons';
+import { Entity } from '@strapi/types';
 import PropTypes from 'prop-types';
 import { useIntl } from 'react-intl';
 
-import useFormatTimeStamp from '../hooks/useFormatTimeStamp';
+import { AuditLog } from '../../../../../../../../shared/contracts/audit-logs';
+import { useFormatTimeStamp } from '../hooks/useFormatTimeStamp';
 import { getDefaultMessage } from '../utils/getActionTypesDefaultMessages';
 
-const TableRows = ({ headers, rows, onOpenModal }) => {
+export type TableHeader = {
```

Comment:
I would take this from the ListView types just merged, it _should_ be a lot better & hopefully will help you type the cell stuff too – let me know if you run into difficulties though!